packages/mobile/src/utils/challenges.tsx

```diff
@@ -0,0 +1,252 @@
+import {
```

Comment:
This really doesn't feel like a "util" but I saw your PR description - probably ok to leave for now and move to "common" somewhere later?

Semi relatedly, could also see grouping "challenges page" stuff (like the config here and the drawers too) in the audio-screen folder? It just seems too far away/spread out from where it's used/useful

Reply:
the drawer gets mounted at the root level though, so it feels kind of weird for it to depend on audio-screen for me. I think a shared web + mobile config needs to go in common/challenges/config.ts, potentially as a class where mobile + web can construct it with ReactNode's for the corresponding/correct images